src/joint.dia.element.js

```diff
@@ -1009,7 +1013,7 @@ joint.dia.ElementView = joint.dia.CellView.extend({
 
             var grid = this.paper.options.gridSize;
             var interactive = _.isFunction(this.options.interactive)
-                ? this.options.interactive(this, 'pointermove')
+                ? this.options.interactive(this)
```

Comment:
Can we introduce here a new feature e.g. `elementMove` and simply use `fi (this.can('elementMove')) { /* move element */ }`?